README.md

```diff
@@ -23,54 +36,46 @@ This package contains transformation models, designed to work simultaneously wit
 | [iterable__list_user_history](https://github.com/fivetran/dbt_iterable/blob/main/models/iterable__list_user_history.sql)             | Each record represents a unique user-list combination. This is intended to recreate the `LIST_USER_HISTORY` source table, which can be disconnected from your syncs, as it can lead to excessive MAR usage. |
```

Comment:
Can you update these links to reference the dbt docs site instead of the models in the repo.